style.css

```diff
@@ -0,0 +1,7 @@
+.header {
+    color: pink;
+}
+
+body {
+    background-color: beige;
```

Comment:
- [ ] As a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should be sure that there are [no linter errors](https://github.com/microverseinc/linters-config) :+1: . I recommend you check the indentation of lines 2 and 6, and also left an empty line after all the elements :+1: .